spec/controllers/book_suggestions_controller_spec.rb

```diff
@@ -14,10 +14,16 @@
       it 'responses with a json' do
         expect(response.content_type).to eq 'application/json'
       end
+
+      it 'creates a new book suggestion' do
+        expect do
+          post :create, params: { book_suggestion: book_suggestion }
+        end.to change { BookSuggestion.count }.by(1)
+      end
     end
 
     context 'when posting an invalid book suggestion' do
-      let(:book_suggestion) { attributes_for(:book_suggestion).merge(year: 'invalid') }
+      let(:book_suggestion) { attributes_for(:book_suggestion, year: 'dsaf', title: nil) }
```

Comment:
Use faker to this false year
```suggestion
      let(:book_suggestion) { attributes_for(:book_suggestion, year: Faker::TvShows::Simpsons.character, title: nil) }
```